spec/instant.html

```diff
@@ -64,8 +64,8 @@ <h1>Temporal.Instant.fromEpochSeconds ( _epochSeconds_ )</h1>
       <emu-alg>
         1. Set _epochSeconds_ to ? ToNumber(_epochSeconds_).
         1. Set _epochSeconds_ to ? NumberToBigInt(_epochSeconds_).
-        1. Let _epochNanoseconds_ be _epochSeconds_ × 1,000,000,000.
-        1. Perform ? RejectInstant(_epochNanoseconds_).
+        1. Let _epochNanoseconds_ be _epochSeconds_ × *1,000,000,000*<sub>ℤ</sub>.
```

Comment:
the spec doesn't currently use any `,` separators in long numbers (like 1000).
```suggestion
        1. Let _epochNanoseconds_ be _epochSeconds_ × *1000000000*<sub>ℤ</sub>.
```

Reply:
OK, in that case I've gone with scientific notation — I'd assume we don't want people to make mistakes counting the zeroes in 3600000000000.